src/notificaciones/presentation/commands.py

```diff
@@ -0,0 +1,72 @@
+from firebase_admin import credentials, messaging
+from flask import Blueprint, request
+from flask_jwt_extended import get_jwt_identity, jwt_required
+
+import seedwork.presentation.api as api
+
+
+bp_prefix: str = "/notificaciones/commands"
+bp: Blueprint = api.create_blueprint("commands", bp_prefix)
+
+
+def subscribe_topic(tokens, topic):  # tokens is a list of registration tokens
+    #   tokens = [
+    #     'YOUR_REGISTRATION_TOKEN_1',
+    #     # ...
+    #     'YOUR_REGISTRATION_TOKEN_n',
+    # ]
+    response = messaging.subscribe_to_topic(tokens, topic)
+    if response.failure_count > 0:
+        print(
+            f"Failed to subscribe to topic {topic} due to {list(map(lambda e: e.reason,response.errors))}"
+        )
+    print(response.success_count, "tokens were subscribed successfully")
```

Comment:
Debería haber un comando para esto
```suggestion
def subscribe_topic(tokens, topic):  # tokens is a list of registration tokens
    response = messaging.subscribe_to_topic(tokens, topic)
    if response.failure_count > 0:
        print(
            f"Failed to subscribe to topic {topic} due to {list(map(lambda e: e.reason,response.errors))}"
        )
    print(response.success_count, "tokens were subscribed successfully")
```